frontend/app/routes/$lang+/_public+/apply+/$id+/adult/review-information.tsx

```diff
@@ -281,10 +281,12 @@ export default function ReviewInformation() {
   const isSubmitting = fetcher.state !== 'idle';
   const { captchaRef } = useHCaptcha();
 
+  const [isBackAction, setIsBackAction] = useState(false);
```

Comment:
Only display spinner if it's action "submit".

```suggestion
  const [isActionSubmit, setIsActionSubmit] = useState(false);
```